# set_speed_1.msg
string hand_name
int32[] finger_ids   
int32[] speeds      # （0-1000）

